05-lists-and-dictionaries/capital_city_loop.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+from capitals import capitals_dict
+from random import choice
+
+def get_random_state():
+    return choice(list(capitals_dict.keys()))
+
+def capital_of_state_quiz(state):
+    while True:
+        response = input("What is the capital of " + state + "? ").lower()
+
+        if response == capitals_dict[state].lower():
+            print("Correct")
+            return True
+        elif response == "exit":
+            print("Goodbye")
+            exit(0)
```

Comment:
Return a value rather than exiting from the function (I prefer that the exit will happen only from `main`).